src/errors/Messages.js

```diff
@@ -98,8 +98,9 @@ const Messages = {
 
   EMOJI_TYPE: 'Emoji must be a string or GuildEmoji/ReactionEmoji',
   EMOJI_MANAGED: 'Emoji is managed and has no Author.',
-  MISSING_MANAGE_EMOJIS_PERMISSION: guild =>
-    `Client must have Manage Emoji permission in guild ${guild} to see emoji authors.`,
+  MISSING_MANAGE_EMOJIS_AND_STICKERS_PERMISSION: guild =>
+    `Client must have Manage Emojis and Stickers permission in guild ${guild} to see emoji authors.`,
```

Comment:
```suggestion
    `Client must have Manage Emojis and Stickers permission in guild ${guild} to see the sticker author.`,
```

Reply:
this error is only used by GuildEmoji#fetchAuthor, I just renamed it bc the permission was renamed

Reply:
Ah, gotcha!